res/values/setup.xml

```diff
@@ -26,17 +26,6 @@
     <string name = "send_files_to_other_apps">on</string>
     <string name = "share_feature">on</string>
     
-    
-    <!-- Colors -->
-    <color name="login_background_color">#FFFFFF</color>
```

Comment:
Can you comment on why this is? It works perfectly fine for me since it is referenced to by type not resource file.